be/src/runtime/load_stream_writer.cpp

```diff
@@ -164,38 +167,99 @@ Status LoadStreamWriter::close_segment(uint32_t segid) {
     return Status::OK();
 }
 
+Status LoadStreamWriter::close_inverted_index(uint32_t segid) {
```

Comment:
It's almost the same as close_segment() and can be merged with file_type argument.

Reply:
And change name to close_writer()

Reply:
done